src/mumble/GlobalShortcut_win.cpp

```diff
@@ -104,22 +127,11 @@ GlobalShortcutEngine *GlobalShortcutEngine::platformInit() {
 	return new GlobalShortcutWin();
 }
 
-
 GlobalShortcutWin::GlobalShortcutWin()
-	: pDI(nullptr), hhMouse(nullptr), hhKeyboard(nullptr), uiHardwareDevices(0)
-#ifdef USE_GKEY
-	  ,
-	  gkey(nullptr)
-#endif
 #ifdef USE_XBOXINPUT
-	  ,
-	  xboxinput(nullptr), nxboxinput(0)
+	: m_xinputDevices(0), m_xinputLastPacket()
```

Comment:
Why the change from `nullptr` to literal `0`. Is the type of `m_xinputDevices` not a pointer?

Reply:
No, it's the number of XInput devices.

`xboxinput` was a raw pointer, it's now a `std::unique_ptr`.